test/powershell/engine/Help/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -8,6 +8,8 @@
     # (when calling get-help -online) might not matched the one in the csv file.
 
     BeforeAll {
+        $SavedProgressPreference = $ProgressPreference
```

Comment:
I think this is a Pester issues, but this pattern can result in the preference not being set back to the original setting.  What should we do about this:

- File an issue with Pester
- Use another pattern
- Live with the issue
- Some other option

# Example Code:
```
PS C:\Users\tplunk> Describe 'before all after all' {
    BeforeAll{
        Write-Verbose "in BeforeAll" -Verbose
        
    }
    it "it" {
        Start-Sleep -Seconds 2

    }
    AfterAll {
        Write-Verbose "in AfterAll" -Verbose
        
    }
}
```

# Example hitting ctl-c during it:

> Describing before all after all
> VERBOSE: in BeforeAll
> VERBOSE: in it
**Note:** After all was not run

# Example of full run

>Describing before all after all
>VERBOSE: in BeforeAll
>VERBOSE: in it
> [+] it 2.13s
>VERBOSE: in AfterAll